gapic_plugin.py

```diff
@@ -157,10 +157,17 @@ def main(data):
   return output
 
 if __name__ == '__main__':
+  try:
```

Comment:
I am curious why this change was necessary. `sys.stdin.read()` and `sys.stdout.write()` are valid in Python 3.